### ISSUE TITLE
fix: Cannot set property 'acebase_origin' of null

### ISSUE THREAD

**Issue by KesleyDavid** [appy-one/acebase-client#14]
I'm using the script importing via the link:
`<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/acebase-client@latest/dist/browser.min.js"></script>`

since the last update yesterday (10/27/2021), when I do a `db.query().get()` query everything goes fine, but when I do a `db.ref().get()` query, I always get the same error

The data is returned if I query the packages in the google crhome "network", but an error appears when doing the query and I can't get the data through javascript.

![image](https://user-images.githubusercontent.com/39314443/139280770-cb64465d-6389-4e44-b03a-4907f9145338.png)

My package:
```json
{
  "scripts": {
    "start": "node server-database.js",
    "dev": "nodemon server-database.js",
    "database": "pm2 start server-database.js",
    "stop": "pm2 stop server-database && pm2 delete server-database"
  },
  "dependencies": {
    "acebase-server": "^1.4.0",
    "nodemon": "^2.0.12"
  }
}

```

I have a hospital client who uses the database, but the entire system is paralyzed, without being able to carry out requests for exams and consultations of reports and images.

I thank you and want to congratulate you on the great database, we are really enjoying using it.

I believe this is a simple bug to fix, and soon you will be able to :D

**Comment by KesleyDavid** [comment]
Good morning @appy-one ,

I updated `acebase-server` to `1.5.1`
I'm using CNN @lasted for `acebase-client`

For me the same error keeps appearing, follow the lines for the errors

![image](https://user-images.githubusercontent.com/39314443/139495002-44d3e71e-a659-4a67-90b0-cb7ff16f40d9.png)

Line Errors:
```
if (!useCache) {
        const { value, context } = await getServerValue();
        context.acebase_origin = 'server';
        return { value, context };
}
```

**Comment by appy-one** [comment]
Maybe you are using a cached version because of a proxy server in between? Try replacing @latest with @1.6.2, or @1.7.1 (latest)

**Comment by appy-one** [comment]
I see there is another issue that I need to fix, will do that first thing tomorrow.

**Comment by appy-one** [comment]
@KesleyDavid I published acebase-client version 1.7.2 just now, let me know if it all works

**Comment by KesleyDavid** [comment]
Perfect, everything working perfectly. Thank you very much @appy-one
